parser-library/nt-headers.h

```diff
@@ -55,6 +55,49 @@ const std::uint16_t DIR_IAT = 12;
 const std::uint16_t DIR_DELAY_IMPORT = 13;
 const std::uint16_t DIR_COM_DESCRIPTOR = 14;
 
+// Machine Types
```

Comment:
TIL about the linkage of const globals. Yeah lets go `constexpr` and call it a PR :-D